Decred Wallet/Features/More/More Features/Settings/SettingsController.swift

```diff
@@ -171,12 +180,45 @@ class SettingsController: UITableViewController  {
             isWalletOpen = true
         }
         
+        let localAuthenticationContext = LAContext()
+        var authError: NSError?
+        var isBiometricSupported = false
+        
+        if localAuthenticationContext.canEvaluatePolicy(LAPolicy.deviceOwnerAuthenticationWithBiometrics, error: &authError) {
+            if #available(iOS 11.0, *) {
+                switch localAuthenticationContext.biometryType {
+                case .faceID:
+                    isBiometricSupported = true
+                    self.biometric_type.text = LocalizedStrings.useFaceID
+                    break
+                    
+                case .touchID:
+                    isBiometricSupported = true
+                    self.biometric_type.text = LocalizedStrings.useTouchId
+                break
+                    
+                case .none:
+                    isBiometricSupported = false
+                break
+                    
+                @unknown default:
+                    isBiometricSupported = false
+                    break
+                }
+            } else {
+                isBiometricSupported = false
+            }
+        }
```

Comment:
Similar code is found in the `StartScreenViewController` class. Refactor into a shared utility function, perhaps a `Utils.availableBiometricsOption` or `Utils.supportedBiometricsOption` that returns a `BiometricsOption` result which would be an enum with values `.none, .fingerprint, .faceID`.